ctp2_code/gs/gameobj/ArmyData.cpp

```diff
@@ -5891,6 +5892,9 @@ ORDER_RESULT ArmyData::InterceptTrade()
 						g_selected_item->ForceDirectorSelect(Army(m_id));
 						return ORDER_RESULT_ILLEGAL;
 					}
+					if (g_player[g_selected_item->GetVisiblePlayer()] == route_owner){ // set pirating sound if route_owner is human player
```

Comment:
Visual Studio 2017 refuses to compile this line. It should only be:
`g_selected_item->GetVisiblePlayer() == route_owner`

Reply:
Many thanks for testing on Win and pointing this out (odd that it works with gcc on linux).
Fixed with [2f0714d](https://github.com/civctp2/civctp2/pull/386/commits/2f0714dc3591adfdf58b79357122274a50566c70)